src/bandersnatch_filter_plugins/blacklist_name.py

```diff
@@ -31,20 +35,15 @@ def _determine_filtered_package_names(self):
         filtered_packages = set()
         try:
             lines = self.configuration['blacklist']['packages']
-            logger.debug('blacklist->packages: %r', lines)
             package_lines = lines.split('\n')
         except KeyError:
             package_lines = []
         for package_line in package_lines:
```

Comment:
NIT newline here is nice so it looks seperate to the try/except - Won't block on it.